template.yaml

```diff
@@ -90,6 +90,8 @@ Resources:
       Role: !GetAtt TriggerExecutionRole.Arn
       MemorySize: 1408
       Timeout: 10
+      ProvisionedConcurrencyConfig:
+        ProvisionedConcurrentExecutions: 3
```

Comment:
``` 
ProvisionedConcurrencyConfig

    The provisioned concurrency configuration of a function's alias.

    Note: ProvisionedConcurrencyConfig can only be specified if the AutoPublishAlias is set. Otherwise, an error results.

    Type: ProvisionedConcurrencyConfig

    Required: No
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-provisionedconcurrencyconfig

    AWS CloudFormation compatibility: This property is passed directly to the ProvisionedConcurrencyConfig property of an AWS::Lambda::Alias resource.
```

Reply:
Not sure if it is going to work as it is now. Can you create a new Pipeline to test it?

Reply:
FYI @axthosarouris : Required permissions etc for using `AutoPublishAlias` https://github.com/BIBSYSDEV/NVA-infrastructure/pull/55